pymc/model/core.py

```diff
@@ -400,76 +393,99 @@ class Model(WithMemoization, metaclass=ContextMeta):
     name : str
         name that will be used as prefix for names of all random
         variables defined within model
+    coords: dict
+        Xarray-like coordinate keys and values. These coordinates can be used
+        to specify the shape of random variables and to label (but not specify)
+        the shape of Determinsitic, Potential and Data objects.
+        Other than specifying the shape of random variables, coordinates have no
+        effect on the model. They can't be used for label-based broadcasting or indexing.
+        You must use numpy-like operations for those behaviors.
     check_bounds : bool
         Ensure that input parameters to distributions are in a valid
         range. If your model is built in a way where you know your
         parameters can only take on valid values you can set this to
         False for increased speed. This should not be used if your model
         contains discrete variables.
+    model : PyMC model, optional
+        A parent model that this model belongs to. If not specified and the current model
+        is created inside another model's context, the parent model will be set to that model.
+        If `None` the model will not have a parent.
 
     Examples
     --------
-    How to define a custom model
+    Use context manager to define model and respective variables
 
     .. code-block:: python
+        import pymc as pm
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+
+    Use object API to define model and respective variables
+
+    .. code-block:: python
+        import pymc as pm
+
+        model = pm.Model()
+        x = pm.Normal("x", model=model)
+
+
+    Use coords for defining the shape of random variables and labeling other model variables
+
+    .. code-block:: python
+    import pymc as pm
+    import numpy as np
+
+    coords = {"feature", ["A", "B", "C"], "trial", [1, 2, 3, 4, 5]}
```

Comment:
```suggestion
    coords = {
        "feature", ["A", "B", "C"],
        "trial", [1, 2, 3, 4, 5],
    }
```
a little easier to read